ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java

```diff
@@ -80,8 +80,8 @@ public boolean isDunkerNoteDetected(){
    * @return true if the note is detected
    */
   public boolean isShooterNoteDetected(){
-    if (shooterBeambreakSensor != null){
-      return shooterBeambreakSensor.isNoteDetected();
+    if (firstShooterBeambreakSensor != null){
```

Comment:
for this method, we want to return true if either tof is true. 
if both are installed return firstShooterBeamBreakSensor.isNoteDetected() OR secondShooterBeamBreakSensor.isNoteDetected() 
If first is installed, return firstShooterBeamBreakSensor.isNoteDetected()
if second is installed, return
secondShooterBeamBreakSensor.isNoteDetected()
else return false.

Reply:
updated method